transport/wseb/src/test/scripts/org/kaazing/gateway/transport/wseb/closeDownstreamShouldUnbindUpstream.rpt

```diff
@@ -186,8 +186,13 @@ write notify UPSTREAM_CLOSE_SENT
 #TIME SEQ: 8 - upstream should be unbound
 read await UPSTREAM_CLOSE_SENT
 read "HTTP/1.1 404 Not Found\r\n"
-read "Content-Type: text/html\r\n"
 read /Content-Length: .*/ "\r\n"
```

Comment:
Syntax like this will cause test failure if there is fragmentation. See the email I just sent and [k3po issue #200](https://github.com/k3po/k3po/issues/200). Need to replace with
```
read /Content-Length: .*\r\n/
```
Same goes the the other lines using that style of syntax.